data/AbhayKumarMittal.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Abhay Kumar Mittal",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open Source | Devops | Full Stack Developer,
```

Comment:
and there's a missing quotation mark at the end of line 5, before the comma

Reply:
Sure , I will correct it.